CMakeLists.txt

```
@@ -99,6 +99,27 @@ if(JACK)
   install(FILES cmake/modules/FindJACK.cmake DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/portaudio/modules")
 endif()
 
+# PulseAudio is mostly for Linux
+
+find_package(PULSEAUDIO)
+cmake_dependent_option(PULSEAUDIO "Enable support for PulseAudio general purpose sound server" ON PULSEAUDIO_FOUND OFF)
+if(PULSEAUDIO)
+  target_link_libraries(PortAudio PRIVATE ${PULSEAUDIO_LIBRARIES})
+  target_sources(PortAudio PRIVATE src/hostapi/pulseaudio/pa_linux_pulseaudio_block.c
+    src/hostapi/pulseaudio/pa_linux_pulseaudio.c
+    src/hostapi/pulseaudio/pa_linux_pulseaudio_cb.c)
+
+  set(PORTAUDIO_PUBLIC_HEADERS "${PORTAUDIO_PUBLIC_HEADERS}" src/hostapi/pulseaudio/pa_linux_pulseaudio_block_internal.h
+    src/hostapi/pulseaudio/pa_linux_pulseaudio_internal.h
+    src/hostapi/pulseaudio/pa_linux_pulseaudio_cb_internal.h)
+
+  target_compile_definitions(PortAudio PRIVATE PA_USE_PULSEAUDIO=1)
+  set(PKGCONFIG_REQUIRES_PRIVATE "${PKGCONFIG_REQUIRES_PRIVATE} pulseaudio")
+
+  # needed for PortAudioConfig.cmake so `find_package(PortAudio)` works in downstream projects
+  install(FILES cmake/modules/FindPULSEAUDIO.cmake DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/portaudio/modules")
```

```suggestion
  install(FILES cmake/modules/FindPulseAudio.cmake DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/portaudio/modules")
```
Please change this back to its original name. JACK is all capitalized because JACK is an acronym.

---

Actually CMakeLint success them to be uppercase I changed because of that.

---

I have not heard of CMakeLint but there is no requirement to use that.

---

It's wonderful Python 3.x tool to manage CMake files. It can be found: https://github.com/cmake-lint/cmake-lint but I'll revert this.

---

But this should be resolved